src/handlers/miner_hbbft_handler.erl

```diff
@@ -698,6 +734,49 @@ kvl_set(K, V, KVL) ->
 b58(<<Bin/binary>>) ->
     libp2p_crypto:bin_to_b58(Bin).
 
+%% do nothing if we've advanced
+process_skips(Proposed, SenderRound, F, Sender, Votes) ->
+    Votes1 = Votes#{Sender => {Proposed, SenderRound}},
```

Comment:
This ensures that each actor can only have one vote at a time, right? Are there protections against an actor changing its vote, or sending different votes to different peers or is this not a risk?